### PR TITLE
Updates to mule.js as per Todd's recommendations

### PR DESCRIPTION
**Pull request by KnightDoom**

### CODE REVIEW

**Comment by Nightfirecat** [comment]
Holy extra commits, Batman! Can you explain the source of the renders/new item information?

File: lib/mule.js [hunk 1]
```diff
@@ -113,11 +113,26 @@ function mulelink(guid) {
 	return l;
 }
 
-var VAULTORDER = [34, 32, 30, 28, 31, 33, 35,
-                  27, 25, 21, 20, 22, 26, 29,
-                  23, 17, 13, 11, 14, 18, 24,
-                  16,  9,  6,  4,  7, 10, 19,
-                  12,  5,  2,  1,  3,  8, 15];
+var VAULTORDER = [
```
Comment (Nightfirecat) [inline lib/mule.js]: I have no problem with this being updated, but unless there's a change in the logic behind generation order, wouldn't this look notably different than ingame?

File: lib/realmapi.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ function update_counter() {
 
 function realmAPI(path, opts, extraopts, callback) {
 	opts.ignore = Math.floor(1e3 + 9e3 * Math.random())
-	var url = BASEURL + path + '?' + $.param(opts)
+	 var url = BASEURL + path + '?' + $.param(opts)+'&muleDump=true'
```
Comment (Nightfirecat) [inline lib/realmapi.js]: Space at beginning of line.

File: updateinstructions.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# How to Update Muledump
```
Comment (Nightfirecat) [inline updateinstructions.md]: Seems fine, but maybe better to be added to README, or linked from it at the very least?

File: lib/constants.js [hunk 4]
```diff
@@ -1328,7 +1328,152 @@ items = {
...
+0x178d: ["Romeo Assassin Skin", 10, -1, 240, 2280, 0, 5000],
+0x178b: ["Juliet Trickster Skin", 10, -1, 280, 2280, 0, 5000],
+0x178c: ["Heartseeker Huntress Skin", 10, -1, 320, 2280, 0, 5000],
+0x178f: ["Carnivalmancer Skin", 10, -1, 360, 2280, 0, 5000],
+0x183d: ["Kageboshi", 25, 0, 400, 2280, 5, 800],
+0x183b: ["Wand of the Fallen", 8, 0, 440, 2280, 5, 650],
+0x899:  ["Dagger of the Hasteful Rabbit", 2, 0, 480, 2280, 3, 310],
+0x8a8:  ["Helm of the Swift Bunny", 16, 0, 520, 2280, 5, 700],
+0x8a9:  ["Vitamine Buster", 11, 0, 560, 2280, 4, 700], 
+//0x13c:  ["Testing Mystery Dye", 10, -1, 760, 2120, 0, 5000], (Unknown)
```
Comment (Nightfirecat) [inline lib/constants.js]: ???

**Comment by KnightDoom** [comment]
Hello Nightfire,
Sorry this was an unintentional PR from atomizer's master. I had intended to do a fork but was new to github. 

There are several missing item images from the inital.The source will have to be provided by thestdman.

File: lib/mule.js (hunk 1 above)
Comment (tischepe) [inline lib/mule.js]: No, I checked (at least for the first 44 chests) this is how it looks ingame. there are now 3 blocks: 2 5x6 and one 5x4 in the middle.

Comment (Nightfirecat) [inline lib/mule.js]: Ok. I haven't played in some time, so I have no idea what the vault situation looks like now...

File: updateinstructions.md (hunk 3 above)
Comment (thestdman) [inline updateinstructions.md]: I have been meaning to but I usually just tell people to redownload the whole thing due to the various different things that must be changed from the master version

**Comment by Nightfirecat** [comment]
@KnightDoom If this isn't meant to be a proper PR, could you close this then? You should be able to maintain your fork regardless of this.

**Comment by Nightfirecat** [comment]
Thanks! :+1:

**Pull request closed without merging**